### PR TITLE
adding Brayns' plugins

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

**Review by pramodk — COMMENTED** [review]
Quick comments: 

- I wouldn't depend on develop for re-producibility.
- instead of commit, I would suggest to use tags/releases

File: var/spack/repos/builtin/packages/brayns-topologyviewer/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from spack import *
+
+
+class BraynsTopologyviewer(CMakePackage):
+
+    git = "ssh://bbpcode.epfl.ch/viz/Brayns-UC-TopologyViewer.git"
+
+    generator = 'Ninja'
+
+    version('0.1.0', commit='58a61253c1e33aea7248c7894398035a36608674', submodules=True,  preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brayns-topologyviewer/package.py]:
```suggestion
    version('0.1.0', commit='58a61253c1e33aea7248c7894398035a36608674', submodules=True)
```

**Review by matz-e — CHANGES_REQUESTED** [review]

**Comment by ppodhajski** [comment]
retest this please

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/brayns-brainatlas/package.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class BraynsBrainatlas(CMakePackage):
+    """Brayns plugin"""
+    git = "ssh://bbpcode.epfl.ch/viz/Brayns-UC-BrainAtlas.git"
+
+    generator = 'Ninja'
+
+    version('develop')
+    version('0.1.0', tag='v0.1.0', preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brayns-brainatlas/package.py]:
```suggestion
    version('0.1.0', tag='v0.1.0')
```

File: var/spack/repos/builtin/packages/brayns-topologyviewer/package.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+class BraynsTopologyviewer(CMakePackage):
+    """Brayns use-case plugin for interactive visualization of
+       the topology of neuron simulation data"""
+
+    git = "ssh://bbpcode.epfl.ch/viz/Brayns-UC-TopologyViewer.git"
+
+    generator = 'Ninja'
+
+    version('develop', submodules=True)
+    version('0.1.0', tag='v0.1.0', submodules=True, preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brayns-topologyviewer/package.py]:
```suggestion
    version('0.1.0', tag='v0.1.0', submodules=True)
```

**Review by matz-e — CHANGES_REQUESTED** [review]
Overlooked this

**Review by matz-e — APPROVED** [review]

**Pull request merged**